### ISSUE TITLE
RuntimeWarning: 'scenedetect.__main__' found in sys.modules after import of package 'scenedetect'

### ISSUE THREAD

**Issue by Breakthrough** [Breakthrough/PySceneDetect#181]
Happens when running via `python -m scenedetect` on latest v0.5.3.  Ensure fixed for v0.5.4 release.

Marking as technical item, not bug, as this only appears to generate a warning and does not currently cause side effects.  It should not appear, however, and indicates that part of the code should be refactored to avoid this warning from being generated.

Full console output:

```
$ python -m scenedetect
Usage: __main__.py [OPTIONS] COMMAND1 [ARGS]... [COMMAND2 [ARGS]...]...

  For example:

  scenedetect -i video.mp4 -s video.stats.csv detect-content list-scenes

  Note that the following options represent [OPTIONS] above. To list the
  optional [ARGS] for a particular COMMAND, type `scenedetect help COMMAND`.
  You can also combine commands (e.g. scenedetect [...] detect-content save-
  images --png split-video).

Options:
  -i, --input VIDEO      [Required] Input video file. May be specified
                         multiple times to concatenate several videos
                         together.
[...]

Commands:
  about             Print license/copyright info.
  detect-content    Perform content detection algorithm on input video(s).

[...]

C:\Users\Breakthrough\AppData\Local\Programs\Python\Python38-32\lib\runpy.py:127: RuntimeWarning: 'scenedetect.__main__' found in sys.modules after import of package 'scenedetect', but prior to execution of 'scenedetect.__main__'; this may result in unpredictable behaviour
  warn(RuntimeWarning(msg))
```

**Comment by Breakthrough** [comment]
See https://stackoverflow.com/a/45070583 for solution.

**Comment by Breakthrough** [comment]
Resolved in v0.5.5.
